developer/src/kmcmpdll/NamedCodeConstants.cpp

```diff
@@ -237,26 +249,28 @@ const int
  HangulNCount = HangulVCount * HangulTCount,   // 588
  HangulSCount = HangulLCount * HangulNCount;   // 11172
 
-const wchar_t *
```

Comment:
@mcdurdin What's special about Hangul that we treat it differently? Why don't we do that for other writing systems like Khmer etc as well?